src/org/jitsi/impl/neomedia/format/MediaFormatImpl.java

```diff
@@ -255,11 +256,11 @@ protected MediaFormatImpl(
         this.formatParameters
             = ((formatParameters == null) || formatParameters.isEmpty())
                 ? EMPTY_FORMAT_PARAMETERS
-                : new HashMap<String, String>(formatParameters);
+                : new ConcurrentHashMap<>(formatParameters);
```

Comment:
That seems unnecessary because the value of the fieldParameters is never modified in and exposed out of MediaFormatImpl.